src/main/java/uk/gov/hmcts/et/common/model/ccd/types/RespondNotificationType.java

```diff
@@ -37,5 +36,6 @@ public class RespondNotificationType {
     private String respondNotificationFullName;
     @JsonProperty("respondNotificationPartyToNotify")
     private String respondNotificationPartyToNotify;
-
+    private String state;
```

Comment:
Would prefer if 'state' was slightly more descriptive so its not confused with other state values. What you can do is try to make sure where you pass the state value to a function in the frontend(without an object reference), you rename it to 'notificationState' or 'repondNotificationState' etc.